src/main.ts

```diff
@@ -15,6 +16,8 @@ async function bootstrap() {
 
   const { port } = configService.get<ServerConfig>('server');
 
+  app.useGlobalPipes(new ValidationPipe());
```

Comment:
To have no extra work for setup e2e testing, It's better to move this one to its own module or related module,
You can create RequestModule and set this validation to it, also in the future also any module that depends on to request operation would be placed here.

Reply:
I didn't understand your point here.
what problem will produce if we using global validation pipe?

Reply:
in e2e setup, you should call ValidaitonPipe in each test file
in e2e you just bring AppModule but this validation is defined in `main.ts`.

Reply:
I understood.
I will add ValidationPipe to AppModule provider.

https://stackoverflow.com/questions/59355841/how-to-apply-global-pipes-during-e2e-tests

Reply:
I think we can do it later